app/routes/__index/dns-records/index.tsx

```diff
@@ -97,9 +97,11 @@ export default function DnsRecordsIndexRoute() {
           DNS Records
         </Heading>
         <Text mb="4" mt="2">
-          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
-          been the industry's standard dummy text ever since the 1500s, when an unknown printer took
-          a galley of type and scrambled it to make a type specimen book.
+          This table shows all of your DNS records that you have created. Once you create a new
```

Comment:
"...all of the DNS records that you have created...new record in the table...needs to be spread across DNS servers...The expiration date...6 months after the creation date and you can request a new certificate before the expiry date."